core/src/types/operator/operator.rs

```diff
@@ -1279,6 +1338,30 @@ impl Operator {
     ///
     /// - Deleting a file that does not exist won't return errors.
     ///
+    /// # Options
+    ///
+    /// ## `version`
+    ///
+    /// Set `version` for this `delete` request.
+    ///
+    /// If `versioning` is not enabled, a `delete` request will permanently remove the file.
+    ///
+    /// when `versioning` is enabled, a simple `delete` request won't permanently remove the file,
+    /// it can still be accessed using its version.
+    /// By setting the `version`, OpenDAL will permanently remove the file with the specified version.
```

Comment:
Hi, it's better not to discuss "permanently delete" here. This behavior is determined by the service itself.

Reply:
how about
```
    /// Set `version` for this `delete` request.
    ///
    /// remove a specific version of the given path.
    ///
    /// If the version doesn't exist, OpenDAL will not return errors.
```